test/adaptors/in-memory-articles.test.ts

```diff
@@ -41,7 +42,8 @@ describe('in-memory articles', (): void => {
   it('throws an error if the article is not found', async (): Promise<void> => {
     const articles = new InMemoryArticles();
 
-    await expect(articles.get('_:1')).rejects.toThrow(new ArticleNotFound('_:1'));
+    await expect(articles.get('_:1')).rejects.toBeInstanceOf(ArticleNotFound);
+    await expect(articles.get('_:1')).rejects.toHaveProperty('message', 'Article _:1 could not be found');
```

Comment:
Like with https://github.com/libero/article-store/pull/101#discussion_r356060843, we probably don't want to check the message. But, we should be able to check `toHaveProperty('id', ' _:1')`.